development/nautobot_config.py

```diff
@@ -85,7 +76,7 @@
 CACHES = {
     "default": {
         "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"{REDIS_SCHEME}://{REDIS_HOST}:{REDIS_PORT}/0",
+        "LOCATION": parse_redis_connection(redis_database=0),
         "TIMEOUT": 300,
         "OPTIONS": {
             "CLIENT_CLASS": "django_redis.client.DefaultClient",
```

Comment:
Is `"PASSWORD": REDIS_PASSWORD` on the line below now an undefined variable? Shouldn't it still pull from `os.getenv("NAUTOBOT_REDIS_PASSWORD", "")`?

For that matter, how does the `"PASSWORD"` setting interact with a password set directly on `"LOCATION"`?

Same question for the other config files.

Reply:
Whoops. I forgot to commit the changes to that file.

As for `PASSWORD`, either form is supported in `django-redis`: https://github.com/jazzband/django-redis#configure-as-cache-backend. So we'll just stick w/ the URL.

Reply:
Ah okay. So we should remove `OPTIONS["PASSWORD"]` from the config and template to avoid confusion, since:

> this option does not overwrites the password in the uri, so if you have set the password in the uri, this settings will be ignored.